README.md

```diff
@@ -156,54 +156,56 @@ The output of the `reduce()` method does not need to be a primitive value like a
 `Number` or `String`. Let's consider an example that accumulates array values
 into an `Object`.
 
-Say we want to create a roster of student wizards assigned to each Hogwarts
-house. Our start value might look like this:
+Say we want to create a roster of student artists based on their discipline of
+art for their final showcase. Our start value might look like this:
 
 ```js
 
-const hogwartsHouses = {
-  "Slytherin": [],
-  "Gryffindor": [],
-  "Hufflepuff": [],
-  "Ravenclaw": []
+const showcases = {
+  "Dance": [],
+  "Visual": [],
+  "Music": [],
+  "Theater": [],
+  "Writing": []
 }
 
 ```
 
-Imagine we also have a `sortingHat` object that includes a `houseAssign()` method.
-That method takes the name of a student as its argument and returns the name of
-the house the student should be assigned to. We could call this method directly:
+Imagine we also have a `sortingObject` object that includes a `showcaseAssign()`
```

Comment:
It's a little harder to follow without the background understanding of what the sorting hat does. Maybe call the object `studentSorter` instead to make it a little more explicit.

Reply:
good call!